fdm-core/src/fertilizer.test.ts

```
@@ -30,6 +30,10 @@ import {
     enableFertilizerCatalogue,
 } from "./catalogues"
 import { createId } from "./id"
+import {
+    disableFertilizerCatalogue,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'disableFertilizerCatalogue'. Consider to delete it or rename it.

'disableFertilizerCatalogue' is defined here:


(lint/suspicious/noRedeclare)